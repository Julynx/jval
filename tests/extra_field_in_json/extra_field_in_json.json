{
    "role": "user",
    "name": "Alice",
    "cars": [
        {
            "type": "car",
            "brand": "Toyota",
            "model": "Corolla",
            "year": 2020,
            "extra_field": "should not be here"
        }
    ]
}