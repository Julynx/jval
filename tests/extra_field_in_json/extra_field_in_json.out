Validation error at 'cars[0]': extra key not defined in schema for key 'extra_field'